generators/workspace/templates/plain_initialize_once/deploy/web/50x.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en" style="height: 100%;">
+<head>
+    <meta charset="UTF-8">
+    <title>Error 50X</title>
+    <link href="https://fonts.googleapis.com/css?family=Yantramanav:700,400" rel="stylesheet" type="text/css">
```

Comment:
[According to Google Fonts](https://fonts.google.com/specimen/Yantramanav?preview.text=&preview.text_type=custom&query=yan&sidebar.open=true&selection.family=Yantramanav:wght@400;700), the modern/current import should look like this:

```suggestion
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="https://fonts.googleapis.com/css2?family=Yantramanav:wght@400;700&display=swap" rel="stylesheet"> 
```